KinoPoiskAutomatedTests/PageObjects/DefaultPage.cs

```diff
@@ -0,0 +1,12 @@
+﻿using KinoPoiskAutomatedTests.TestData;
+using KinoPoiskAutomatedTests.WebDriver;
+using OpenQA.Selenium;
+
+namespace KinoPoiskAutomatedTests.PageObjects
+{
+    public class DefaultPage
+    {
+        internal IWebDriver driver = DriverFactory.GetDriver();
```

Comment:
Receiving of current state of WebDriver would much better to move to the DefaultPage constructor.